### ISSUE TITLE
Why GPL 3.0?

### ISSUE THREAD

**Issue by alsms** [parvez3019/go-swagger3#52]
I like this library but why is GPL 3.0 necessary? Isn’t it virtually useless for serious usage this way?

**Comment by parvez3019** [comment]
Hi 

GPLv3 Explained here - https://gist.github.com/kn9ts/cbe95340d29fc1aaeaa5dd5c059d2e60#file-gplv3-md

Can you please briefly explain what issue do you see with GPLv3 on this library?

**Comment by alsms** [comment]
Yes, it’s basically the first comment. There is no way we can use it with this license, even though the library looks nice.

**Comment by parvez3019** [comment]
3. You can use this software privately.
4. You can use this software for commercial purposes.
5.

**Comment by parvez3019** [comment]
I'll update the Licence to MIT licensed.
